samples/Samples.Wcf/Samples.Wcf.csproj

```diff
@@ -0,0 +1,11 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <ApplicationManifest>app1.manifest</ApplicationManifest>
```

Comment:
Consider removing the application manifest unless you needed one and added it intentionally. I think VS adds one automatically when you do certain actions, like opening the manifest properties page.

Reply:
I added it intentionally. I need to figure out a good way to structure the WCF automation I want to put in, but for now, it asks for admin permissions in order to expose the port.